pt-BR/mail/chrome/messenger/importDialog.dtd

```diff
@@ -36,26 +36,26 @@
  ***** END LICENSE BLOCK ***** -->
 
 <!--
-LOCALIZATION NOTE : 'Communicator 4.x' is the used for previous versions of 
-Netscape Communicator, Please translate using the brandname in respective 
+LOCALIZATION NOTE : 'Communicator 4.x' is the used for previous versions of
+Netscape Communicator, Please translate using the brandname in respective
 languages for Netscape Communicator 4 releases.
-LOCALIZATION NOTE : Do not translate any of the occurrences of the word 
+LOCALIZATION NOTE : Do not translate any of the occurrences of the word
 "&brandShortName;" below.
 -->
 
 <!ENTITY importDialog.windowTitle "Assistente de importação">
-<!ENTITY importDialog.allTitle    "Import everything">
-<!ENTITY importDialog.mailTitle   "Mail">
-<!ENTITY importDialog.addressbookTitle "Address Books">
-<!ENTITY importDialog.settingsTitle "Settings">
+<!ENTITY importDialog.allTitle    "Importar tudo">
+<!ENTITY importDialog.mailTitle   "E-mails">
```

Comment:
@brunotiussi Understood. After receiving Sherman's approval, either use e-mail or email (chose one to keep consistency).